src/vs/workbench/services/workingCopy/common/untitledFileWorkingCopy.ts

```diff
@@ -190,7 +192,9 @@ export class UntitledFileWorkingCopy<M extends IUntitledFileWorkingCopyModel> ex
 		await this.doCreateModel(untitledContents);
 
 		// Untitled associated to file path are dirty right away as well as untitled with content
```

Comment:
Should probably update the comment here

Reply:
The check now only exists within `setDirty` to make the usage consistent in the file, but I added a comment in there.